src/app/@ansyn/menu-items/filters/components/filters-collection/filters-collection.component.html

```diff
@@ -1,12 +1,24 @@
 <div class="collection">
-	<ansyn-slider-checkbox [checked]="onlyFavorite"  (checkedChange)="showOnlyFavorites($event)" [disabled]="disableShowOnlyFavoritesSelection">
+	<ansyn-slider-checkbox [checked]="onlyFavorite" (checkedChange)="showOnlyFavorites($event)"
+						   [disabled]="disableShowOnlyFavoritesSelection">
 		Show only favorites
 	</ansyn-slider-checkbox>
 
 	<div class="allFilters">
-			<ansyn-filter-container
+		<ansyn-filter-container
 			*ngFor="let filter of filters"
 			[filter]="filter">
 		</ansyn-filter-container>
+		<div class="removed-show-hide">
```

Comment:
create separated component for this filter

Reply:
: